_posts/release/2017-06-22-fluo-recipes-1.1.0-incubating.md

```diff
@@ -0,0 +1,252 @@
+
+      try (Transaction tx = fc.newTransaction()) {
+        xytCq.addAll(tx, updates);
+        tx.commit();
+      }
+
+      miniFluo.waitForObservers();
+      System.out.println("\n*** All notifications processed. ***\n");
+
+      updates.clear();
+      updates.put("x=3:y=5:t=23", 1L);
+      updates.put("x=5:y=5:t=27", -1L);
+      updates.put("x=3:y=5:t=29", 1L);
+
+      try (Transaction tx = fc.newTransaction()) {
+        xytCq.addAll(tx, updates);
+        tx.commit();
+      }
+
+      miniFluo.waitForObservers();
+      System.out.println("\n*** All notifications processed. ***\n");
+    }
+```
+
+Below is the output of running this example.
+
+```
+EXPORT x=3:y=5:t=23    old: - new: 1   seq: 8
+EXPORT x=3:y=5:t=27    old: - new: 1   seq: 9
+EXPORT x=5:y=5:t=27    old: - new: 1   seq: 9
+EXPORT x=3:y=5         old: - new: 2   seq: 37
+EXPORT y=5:t=27        old: - new: 2   seq: 42
+EXPORT x=3:t=23        old: - new: 1   seq: 36
+EXPORT x=5:t=27        old: - new: 1   seq: 36
+EXPORT x=3:t=27        old: - new: 1   seq: 38
+EXPORT x=5:y=5         old: - new: 1   seq: 39
+EXPORT y=5:t=23        old: - new: 1   seq: 41
+
+*** All notifications processed. ***
+
+EXPORT x=3:y=5:t=29    old: - new: 1   seq: 92
+EXPORT x=5:y=5:t=27    old: 1 new: -   seq: 92
+EXPORT x=3:y=5:t=23    old: 1 new: 2   seq: 93
+EXPORT y=5:t=27        old: 2 new: 1   seq: 109
+EXPORT x=3:y=5         old: 2 new: 4   seq: 110
+EXPORT y=5:t=23        old: 1 new: 2   seq: 111
+EXPORT y=5:t=29        old: - new: 1   seq: 108
+EXPORT x=3:t=29        old: - new: 1   seq: 105
+EXPORT x=3:t=23        old: 1 new: 2   seq: 106
+EXPORT x=5:y=5         old: 1 new: -   seq: 107
+EXPORT x=5:t=27        old: 1 new: -   seq: 106
+
+*** All notifications processed. ***
+```
+
+
+[procedures]: https://www.apache.org/info/verification
+[KEYS]: https://www.apache.org/dist/incubator/fluo/KEYS
+[src-release]: https://www.apache.org/dyn/closer.lua/incubator/fluo/fluo-recipes/1.1.0-incubating/fluo-recipes-1.1.0-incubating-source-release.tar.gz
+[src-asc]: https://www.apache.org/dist/incubator/fluo/fluo-recipes/1.1.0-incubating/fluo-recipes-1.1.0-incubating-source-release.tar.gz.asc
+[md5]: https://www.apache.org/dist/incubator/fluo/fluo-recipes/1.1.0-incubating/fluo-recipes-1.1.0-incubating-source-release.tar.gz.md5
+[sha]: https://www.apache.org/dist/incubator/fluo/fluo-recipes/1.1.0-incubating/fluo-recipes-1.1.0-incubating-source-release.tar.gz.sha
+[docs]: /docs/fluo-recipes/1.1.0-incubating
+[central]: http://search.maven.org/#search|ga|1|fluo-recipes
+[changes]: https://github.com/apache/incubator-fluo-recipes/milestone/1?closed=1
+[obsAPI]: /release/fluo-1.1.0-incubating/#new-api-for-configuring-observers
+[ObserverProvider]: {{ site.fluo_api_static }}/1.1.0-incubating/org/apache/fluo/api/observer/ObserverProvider.html
+[CombineQueue]: {{ site.fluo_recipes_core_static }}/1.1.0-incubating/org/apache/fluo/recipes/core/combine/CombineQueue.html
```

Comment:
Yeah, when I run jekyll it resolves to the following link which I think is correct.  However the link is not currently working at javadoc.io.  This is because I tried to access the link before the artifacts were published to maven central.  Now javadoc.io seems to be caching the fact that the artifacts were not present.  Hopefully this cache times out, otherwise will need to contact them.

https://javadoc.io/page/org.apache.fluo/fluo-recipes-core/1.1.0-incubating/org/apache/fluo/recipes/core/combine/CombineQueue.html